zio-flow/shared/src/main/scala/zio/flow/Remote.scala

```diff
@@ -3156,6 +3288,9 @@ object Remote {
       .:+:(Tuple22.schemaCase[A])
       .:+:(TupleAccess.schemaCase[A])
       .:+:(Branch.schemaCase[A])
+      .:+:(StringToList.schemaCase[A])
+      .:+:(ListToString.schemaCase[A])
+      .:+:(CharAt.schemaCase[A])
```

Comment:
You did not add `Substring`. Please add the new constructors to the serialization test suite too